package.json

```diff
@@ -22,11 +22,11 @@
   "main": "lib/index.js",
   "scripts": {
     "testSpecified": "tap --reporter spec --no-coverage --color --timeout 10 --bail --jobs-auto",
-    "testWithNode": "for file in `find test -name *.js`; do node \"$file\" || exit 1; done;",
-    "test": "npm run testSpecified test",
-    "makeTestSnapshots": "TAP_SNAPSHOT=1 npm run testSpecified test",
-    "makeCodeCoverageDetailReport": "npm run testSpecified test -- --reporter dot --coverage --coverage-report html --no-browser test && echo \"*** Code coverage report done, checkout ./coverage dir\"",
-    "makeCodeCoverageSummaryReport": "npm run testSpecified test -- --reporter dot --coverage --coverage-report text-summary test | sed -re 's/(Statements|Branches|Functions|Lines)/\\1 code coverage/g'",
+    "testWithNode": "for file in `find test -name *.js | grep -v es-modules`; do node \"$file\" || exit 1; done;",
+    "test": "NODE_OPTIONS='--experimental-modules --experimental-specifier-resolution=node --no-warnings' npm run testSpecified test",
+    "makeTestSnapshots": "NODE_OPTIONS='--experimental-modules --experimental-specifier-resolution=node --no-warnings' TAP_SNAPSHOT=1 npm run testSpecified test",
+    "makeCodeCoverageDetailReport": "npm run testSpecified test -- --grep es-modules --invert --reporter dot --coverage --coverage-report html --no-browser test && echo \"*** Code coverage report done, checkout ./coverage dir\"",
+    "makeCodeCoverageSummaryReport": "npm run testSpecified test -- --grep es-modules --invert --reporter dot --coverage --coverage-report text-summary test | sed -re 's/(Statements|Branches|Functions|Lines)/\\1 code coverage/g'",
```

Comment:
There is a bug with coverage and `NODE_OPTIONS='--experimental-modules --experimental-specifier-resolution=node'`, so I decided to skip `es-modules` test for coverage